requirements.txt

```diff
@@ -164,7 +167,7 @@ msal-extensions==1.1.0
     # via azure-identity
 mypy-extensions==1.0.0
     # via typing-inspect
-numpy==1.26.2
+numpy==1.24.4
```

Comment:
We're ok with downgrading Numpy and Pandas here?

Reply:
This is what the dependency resolver came up with 🤷 

I was a little concerned, more with the possibility of introducing a new vulnerability, but `docker scout` scans of the agent images (ran locally) didn't come up with anything new.

Reply:
@walles2 @mrostan any concerns here with the downgraded `numpy` and `pandas`?

Reply:
I think it's fine, data-collector in master is using:
- numpy==1.23.4
- pandas==1.3.5
so, even when we're downgrading we're still using versions after those being used by the dc

Reply:
sorry, assumed it was already approved